setup.py

```diff
@@ -27,12 +29,13 @@
         'License :: OSI Approved :: MIT License',
     ],
     keywords='pydocstyle, PEP 257, pep257, PEP 8, pep8, docstrings',
+    packages=('pydocstyle',),
+    py_modules=('main',),
```

Comment:
The point of this is to allow to run `pydocstyle` from the command line without the entry point (e.g., if you want to just unzip it somewhere and run it without installing).